tests/unit-tests/admin/test-class-sensei-onboarding.php

```diff
@@ -40,6 +40,169 @@ public function testClassInstance() {
 		$this->assertTrue( class_exists( 'Sensei_Onboarding' ), 'Sensei Onboarding class does not exist' );
 	}
 
+	/**
+	 * Test onboarding wizard notice in dashboard.
+	 *
+	 * @covers Sensei_Onboarding::onboarding_wizard_notice
+	 * @covers Sensei_Onboarding::is_current_screen_selected_to_wizard_notice
+	 */
+	public function testOnboardingWizardNoticeInDashboard() {
+		// Create and login as admin.
+		$admin_id = $this->factory->user->create( array( 'role' => 'administrator' ) );
+		wp_set_current_user( $admin_id );
+
+		set_current_screen( 'dashboard' );
+		update_option( \Sensei_Onboarding::SUGGEST_ONBOARDING_OPTION, 1 );
+
+		ob_start();
+		Sensei()->onboarding->onboarding_wizard_notice();
+		$html = ob_get_clean();
+
+		$this->assertNotEmpty( $html, 'Should return the notice HTML' );
+	}
+
+	/**
+	 * Test onboarding wizard notice in screen with Sensei prefix.
+	 *
+	 * @covers Sensei_Onboarding::onboarding_wizard_notice
+	 * @covers Sensei_Onboarding::is_current_screen_selected_to_wizard_notice
+	 */
+	public function testOnboardingWizardNoticeInSenseiScreen() {
+		// Create and login as admin.
+		$admin_id = $this->factory->user->create( array( 'role' => 'administrator' ) );
+		wp_set_current_user( $admin_id );
+
+		set_current_screen( 'sensei-lms_page_sensei_test' );
+		update_option( \Sensei_Onboarding::SUGGEST_ONBOARDING_OPTION, 1 );
+
+		ob_start();
+		Sensei()->onboarding->onboarding_wizard_notice();
+		$html = ob_get_clean();
+
+		$this->assertNotEmpty( $html, 'Should return the notice HTML' );
+	}
+
+	/**
+	 * Test onboarding wizard notice in no Sensei screen.
+	 *
+	 * @covers Sensei_Onboarding::onboarding_wizard_notice
+	 * @covers Sensei_Onboarding::is_current_screen_selected_to_wizard_notice
+	 */
+	public function testOnboardingWizardNoticeInOtherScreen() {
+		// Create and login as admin.
+		$admin_id = $this->factory->user->create( array( 'role' => 'administrator' ) );
+		wp_set_current_user( $admin_id );
+
+		set_current_screen( 'other' );
+		update_option( \Sensei_Onboarding::SUGGEST_ONBOARDING_OPTION, 1 );
+
+		ob_start();
+		Sensei()->onboarding->onboarding_wizard_notice();
+		$html = ob_get_clean();
+
+		$this->assertEmpty( $html, 'Should return empty string' );
+	}
+
+	/**
+	 * Test onboarding wizard notice with suggest option as 0.
+	 *
+	 * @covers Sensei_Onboarding::onboarding_wizard_notice
+	 * @covers Sensei_Onboarding::is_current_screen_selected_to_wizard_notice
+	 */
+	public function testOnboardingWizardNoticeSuggest0() {
```

Comment:
Maybe `testOnboardingWizardNoticeSuggestOptionFalse`?

Reply:
I renamed to `testOnboardingWizardNoticeSuggestOptionAsZero` because it would not be exactly `false`. https://github.com/Automattic/sensei/pull/3096/commits/0a0a552b147c54cc9f9c8f73a09051182764a5d9

Sound good?